src/components/segmentedControl/segment.tsx

```diff
@@ -112,7 +112,7 @@ const Segment = React.memo((props: SegmentProps) => {
     onLayout?.(index, event);
   },
   [onLayout, index]);
-
+  const labelAndIconSpacings = !!iconSource && (iconOnRight ? styles.rightMargin : styles.leftMargin);
```

Comment:
I think you should rename it since the Spacing is going to the Text component.
`labelMargins` or something like that WDYT ?

Reply:
Hmm, I dont see much difference. Your is shorter so we can use that.